var/spack/repos/builtin/packages/py-wasabi/package.py

```diff
@@ -13,6 +13,10 @@ class PyWasabi(PythonPackage):
     homepage = "https://ines.io/"
```

Comment:
According to pypi the homepage is
```suggestion
    homepage = "https://github.com/explosion/wasabi"
```

Reply:
Optional. Always good to update, but shouldn't hold up the PR from being merged.